src/form-elements/FormElementDate.tsx

```diff
@@ -61,7 +61,7 @@ function FormElementDate({
 
   const [commonProps, openDatePicker] = useFormDatePickerProps({
     id,
-    value: typeof value === 'string' ? value : undefined,
+    value: parseDateValue({ dateOnly: false, daysOffset: 0, value }),
```

Comment:
we don't need to do this for the other values?

Reply:
nein, we store them as ISOs already